ScratchWikiSkinTemplate.php

```diff
@@ -103,6 +103,39 @@ public function execute() {
 					</div>
 				</form>
 			</li>
+			<?php
+			//  Start dropdown here so we know its output before it saves to the page, and we can use it earlier.
+			$swsDropdownItems = "";
+			foreach ($this->data['personal_urls'] as $key => $tab) { 
+					if (!array_key_exists('href', $tab)) {
+						if (!array_key_exists('class', $tab)) {
+							$tab['class'] = '';
+						}
+						
+						$tab['class'] .= 'no-link';
+					}
+					if(isset($tab["class"]) && gettype($tab["class"]) == "array" && $tab["class"][0] == "mw-echo-alert"){
+						// Look through the items being outputted for an alert from Echo- this means you're being alerted to check your notifications, regardless of the "unseen" status
+						$swsUnread = true;
+						continue;
+					}
+					if(strpos($key, "notifications") !== false){
+						if(in_array("mw-echo-unseen-notifications", $tab["link-class"])){
+							$swsUnread = true;
+						}
+						continue;
+					}
+					$swsDropdownItems .= $this->getSkin()->makeListItem($key, $tab);
+			}
+			// ONLY output this if Echo extension is turned on. Otherwise don't put it there.
+			if(ExtensionRegistry::getInstance()->isLoaded("Echo")){
+			?>
+			<li class="link right notifications">
+				<a href="<?php echo Title::newFromText("Special:Notifications")->getLocalURL();?>"><div></div></a>
+				<?php echo(wfMessage( 'portal-url' )->inContentLanguage()->text())?>
```

Comment:
What is this doing here? This causes stray text to appear on my machine:
![image](https://github.com/InternationalScratchWiki/ScratchWikiSkin2/assets/1960707/c8dff4ec-0f99-41d3-a0a1-02a953785932)

Reply:
Oops. I added that during testing and forgot to remove it.